### PR TITLE
Survey page addition

### PR DESCRIPTION
**Pull request by aabuderi**
-added arrow to collectionViewCell for juicy UX goodness
-separated the editable fields from the detail view and put in separate page
-formatted the detail view further

### CODE REVIEW

File: src_code/BoundlessBrilliance/Controller/PresentationDetailController.swift [hunk 1]
```diff
@@ -54,22 +54,22 @@ class PresentationDetailController : UIViewController{
     let Grade: UITextView! = {
         let gf = UITextView()
         gf.textColor = UIColor.black
-        gf.text = "Grade: 3"
```
Comment (vinnykris) [inline src_code/BoundlessBrilliance/Controller/PresentationDetailController.swift]: not really clear what gf is... maybe use a more descriptive variable name? using context clues i can figure it out but just a thought

File: src_code/BoundlessBrilliance/Controller/PresentationDetailController.swift [hunk 2]
```diff
@@ -115,6 +115,18 @@ class PresentationDetailController : UIViewController{
         return te
     }()
     
+//    let Pres1Label: UITextView! = {
+//        let te = UITextView()
+//        te.textColor = UIColor.black
+//        te.text = ""
+//        te.isEditable = false
+//        te.translatesAutoresizingMaskIntoConstraints = false
+//        te.textAlignment = NSTextAlignment.right
```
Comment (vinnykris) [inline src_code/BoundlessBrilliance/Controller/PresentationDetailController.swift]: is this comment block needed still?

File: src_code/BoundlessBrilliance/Controller/PresentationDetailController.swift [hunk 3]
```diff
@@ -128,30 +140,42 @@ class PresentationDetailController : UIViewController{
         return p1e
     }()
     
+//    let Pres2Label: UITextView! = {
```
Comment (vinnykris) [inline src_code/BoundlessBrilliance/Controller/PresentationDetailController.swift]: is this comment block needed still?

File: src_code/BoundlessBrilliance/Controller/PresentationDetailController.swift [hunk 4]
```diff
@@ -128,30 +140,42 @@ class PresentationDetailController : UIViewController{
...
-        let exp = UITextField()
-        exp.textColor = UIColor.black
-        exp.text = "Experiment Performed"
-        exp.translatesAutoresizingMaskIntoConstraints = false
-        exp.textAlignment = NSTextAlignment.right
-        exp.backgroundColor = UIColor.white
-        exp.font = .systemFont(ofSize: UIDevice.current.userInterfaceIdiom == .pad ? 15 : 13)
-        return exp
-    }()
+//    let Experiment: UITextField! = {
```
Comment (vinnykris) [inline src_code/BoundlessBrilliance/Controller/PresentationDetailController.swift]: is this comment block needed still?

File: src_code/BoundlessBrilliance/Controller/PresentationDetailController.swift [hunk 5]
```diff
@@ -523,44 +546,40 @@ class PresentationDetailController : UIViewController{
...
         namesField.leftAnchor.constraint(equalTo: NamesFieldTitle.rightAnchor).isActive = true
         
-        Pres1Email.topAnchor.constraint(equalTo: NamesField.bottomAnchor, constant: 6).isActive = true
-        Pres1Email.widthAnchor.constraint(equalTo: inputsView.widthAnchor).isActive = true
-        Pres1Email.heightAnchor.constraint(equalToConstant: 40).isActive = true
-        
-        Pres2Email.topAnchor.constraint(equalTo: Pres1Email.bottomAnchor).isActive = true
-        Pres2Email.widthAnchor.constraint(equalTo: inputsView.widthAnchor).isActive = true
-        Pres2Email.heightAnchor.constraint(equalToConstant: 40).isActive = true
+//        Pres1Email.topAnchor.constraint(equalTo: NamesField.bottomAnchor, constant: 6).isActive = true
```
Comment (vinnykris) [inline src_code/BoundlessBrilliance/Controller/PresentationDetailController.swift]: is this comment block needed still?

File: src_code/BoundlessBrilliance/View/PresentationListCollectionViewCell.swift [hunk 6]
```diff
@@ -50,19 +50,32 @@ class PresentationListCollectionViewCell: UICollectionViewCell {
...
-        $0.titleLabel?.adjustsFontSizeToFitWidth = true
-        $0.titleLabel?.font = UIFont.boldSystemFont(ofSize: 16)
-        $0.layer.cornerRadius = 5
-        return $0
-    }(UIButton())
-
-    //end examples from the alerts and pickers app cell file-----------------------------
+    
+    
+//    public lazy var leaveInputButton: UIButton = {
```
Comment (vinnykris) [inline src_code/BoundlessBrilliance/View/PresentationListCollectionViewCell.swift]: is this comment block needed still?

File: src_code/BoundlessBrilliance/Controller/PresentationDetailController.swift (hunk 1 above)
Comment (aabuderi) [reply src_code/BoundlessBrilliance/Controller/PresentationDetailController.swift]: gf is a UITextView that Grade gets set to at the end of that block. It's kind of unnecessary to create a new UITextView inside that variable so I replaced all names like that with the $0 macro, which is a common Swift macro

**Pull request closed without merging**